control/lib/interaction.js

```diff
@@ -93,7 +93,9 @@ wax.interaction = function() {
                     e: e
                 });
             } else {
-                bean.fire(interaction, 'off');
+                bean.fire(interaction, 'off', error && {
```

Comment:
I'd do the same in line 89 (`bean.fire(interaction, 'on'...`). If I recall correctly, `on` events are triggered when mouse is over a feature and `off` events are triggered when events are NOT passing over a feature. We want to include the error in both cases IMO.

Reply:
But that scenario I think it's not possible. I mean, when there is a limit error, the gridjson doesn't contain any feature, only errors, see the line https://github.com/CartoDB/wax/pull/17/files#diff-296117891555ffc63dc0050305f266fcR112.

Reply:
Makes sense 👍